ftw/upgrade/indexing.py

```diff
@@ -0,0 +1,72 @@
+import pkg_resources
+
+HAS_INDEXING = False
+
+try:
+    # Plone 5
+    from Products.CMFCore.indexing import processQueue
+    from Products.CMFCore.indexing import getQueue
+except ImportError:
+    try:
+        # Plone 4 with collective.indexing
+        pkg_resources.get_distribution('collective.indexing')
+        from collective.indexing.queue import getQueue
+        from collective.indexing.queue import processQueue
```

Comment:
I would move the imports to the `else` part, because we would not want to catch exceptions happing at import time here. See my other comment.